predict-daily-temperature/Temperature-Demo.ipynb

```diff
@@ -0,0 +1,6470 @@
+       "1168                               22.812500   \n",
+       "\n",
+       "      NUMERIC_LAG(date, meantemp, periods=9)  \\\n",
+       "id                                             \n",
+       "16                                  8.857143   \n",
+       "17                                 14.000000   \n",
+       "18                                 11.000000   \n",
+       "19                                 15.714286   \n",
+       "20                                 14.000000   \n",
+       "...                                      ...   \n",
+       "1164                               24.916667   \n",
+       "1165                               24.933333   \n",
+       "1166                               26.000000   \n",
+       "1167                               27.312500   \n",
+       "1168                               23.933333   \n",
+       "\n",
+       "      ROLLING_MEAN(date, meantemp, window_length=6, gap=8, min_periods=6)  \\\n",
+       "id                                                                          \n",
+       "16                                             8.587302                     \n",
+       "17                                             8.976190                     \n",
+       "18                                            10.595238                     \n",
+       "19                                            11.761905                     \n",
+       "20                                            13.234127                     \n",
+       "...                                                 ...                     \n",
+       "1164                                          23.720833                     \n",
+       "1165                                          24.254167                     \n",
+       "1166                                          25.006250                     \n",
+       "1167                                          25.161806                     \n",
+       "1168                                          24.984722                     \n",
+       "\n",
+       "     YEAR(date)  \n",
+       "id               \n",
+       "16         2013  \n",
+       "17         2013  \n",
+       "18         2013  \n",
+       "19         2013  \n",
+       "20         2013  \n",
+       "...         ...  \n",
+       "1164       2016  \n",
+       "1165       2016  \n",
+       "1166       2016  \n",
+       "1167       2016  \n",
+       "1168       2016  \n",
+       "\n",
+       "[1153 rows x 13 columns]"
+      ]
+     },
+     "execution_count": 55,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "X_train = remove_nans(train_fm, target_col)\n",
+    "y_train = X_train.pop(target_col)\n",
+    "\n",
+    "X_train"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4faf5ebc",
+   "metadata": {},
+   "source": [
+    "Now, we'll use the feature definitions from the DFS run on the test data, producing the test feature matrix."
```

Comment:
To a novice user they might feel a little lost as to what is happening here. It could help to provide a more in-depth explanation of what's going on and/or link to definitions and docs!